libbpf-tools/fsdist.h

```diff
@@ -0,0 +1,20 @@
+/* SPDX-License-Identifier: (LGPL-2.1 OR BSD-2-Clause) */
+#ifndef __FSDIST_H
+#define __FSDIST_H
+
+enum fs_file_op {
+	READ,
+	WRITE,
+	OPEN,
+	FSYNC,
+	GETATTR,
+	MAX_OP,
+};
+
+#define MAX_SLOTS	32
```

Comment:
why the bump in number of slots? ext4dist uses 27, is that not enough?

Reply:
For NFS we may need a large value.